flake.nix

```diff
@@ -119,6 +119,8 @@
                   cargo-watch
                   wabt
 
+                  strictdoc
+
                   # utilities
```

Comment:
Is this necessary? I would assume via inputsFrom we already have this

Reply:
Doesn't seem so. The `requirements-*` commands don't work with `strictdoc: command not found`.

Reply:
My mistake. `self.packages.${system}.requirements` is missing from `packagesFrom` in the devshell. Please add it above `self.packages.${system}.whitepaper` to maintain alphanumerical sorting.